### DISCUSSION TITLE
Syntax Highlighter for Elixir

### DISCUSSION THREAD

**Discussion by kamilpikula** [GetPublii/Publii#1736] [Ideas]
Hi, what do you think about adding a new language - [Elixir](https://elixir-lang.org/) - to the existing syntax highlighter? It doesn't seem too hard to do: https://github.com/GetPublii/Publii/commit/5278cb69e4e23c96c6fa199d1865cfa845bfa557

**Comment by dziudek** [comment]
@kamilpikula - will be added in v.0.46.2: https://github.com/GetPublii/Publii/commit/429a1e04bf0daf3e08b50b46506a61ef253e6a2a :)
